hawkpost/templates/account/auth_modals.html

```diff
@@ -70,20 +65,15 @@ <h3 class="title "> Login</h3>
   <div class="md-content-xs ">
     <button type="button" class="md-close btn-none"><i class="fa fa-times fa-lg" aria-hidden="true"></i></button>
     <div class="row-s start-xs ">
-      <form class="signup start-xs smalltext text-darkest" id="signup-form-js" method="post" action="{% url 'account_signup' %}">
+      <form class="signup start-xs smalltext text-lightest" id="signup-form-js" method="post" action="{% url 'account_signup' %}">
         <div class="col-xs-12">
           <h3 class="title"> Sign up</h3>
           {% get_providers as socialaccount_providers %}
           {% if socialaccount_providers %}
```

Comment:
If doesn't make much sense to check for social account providers in order to put the text `Sign up via email address`, please review this logic on both modals.